viewer/tsconfig.json

```diff
@@ -5,7 +5,8 @@
     "module": "esnext",
     "lib": [
       "dom",
-      "esnext"
+      "esnext",
+      "webworker"

```

Comment:
Was this one needed, in the end?

Reply:
Yup, needed for this specific overload apparently: https://github.com/cognitedata/reveal/pull/2321/files#diff-18839a6b13a7d5cc46b9f811597012616d27bcb330c9391ccaba84eab0d9390cR33